src/Core/Utilities/EncodedStringConverter.cs

```diff
@@ -29,7 +37,15 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 return;
             }
 
-            writer.WriteValue(System.Net.WebUtility.HtmlEncode((string)value));
+            RemoveConverterAndAct(serializer, () =>
+            {
+                serializer.Serialize(writer, new EncodedString { v = CoreHelpers.Base64EncodeString((string)value) });
+            });
         }
     }
+
+    public class EncodedString
+    {
+        public string v { get; set; }
```

Comment:
What does v mean here? I don't really know what to name this instead, but that seems weird.